src/main/java/redis/clients/jedis/JedisSocketFactory.java

```diff
@@ -34,4 +34,6 @@
   int getSoTimeout();
 
   void setSoTimeout(int soTimeout);
+
+  JedisSocketFactory copy();
```

Comment:
Why copy and not clone and clonable?

Reply:
Commented above, will resolve this conversation in favour of the above one but whatever we agree will be made here too.